Documentation/Upgrade/ceph-upgrade.md

```diff
@@ -18,41 +18,41 @@ until all the daemons have been updated.
 ## Considerations
 
 * **WARNING**: Upgrading a Rook cluster is not without risk. There may be unexpected issues or
-  obstacles that damage the integrity and health of your storage cluster, including data loss.
+  obstacles that damage the integrity and health of the storage cluster, including data loss.
 * The Rook cluster's storage may be unavailable for short periods during the upgrade process.
-* We recommend that you read this document in full before you undertake a Ceph upgrade.
+* Read this document in full before undertaking a Rook cluster upgrade.
 
 ## Supported Versions
 
 Rook v1.12 supports the following Ceph versions:
 
+* Ceph Reef v18.2.0 or newer
 * Ceph Quincy v17.2.0 or newer
 * Ceph Pacific v16.2.0 or newer
 
-Support for Ceph Octopus (15.2.x) was removed. If you are running Octopus you must upgrade the Ceph version
-before upgrading to Rook v1.12.
+Support for Ceph Pacific (16.2.x) will be in the next Rook release. It will be mandatory to upgrade
+to Quincy or Reef before upgrading to the Rook release after v1.12.x.
 
 !!! important
     When an update is requested, the operator will check Ceph's status,
     **if it is in `HEALTH_ERR` the operator will refuse to proceed with the upgrade.**
 
-We recommend updating to v16.2.7 or newer. If you require updating **to v16.2.0-v16.2.6**,
```

Comment:
Shall we remove this paragraph about 16.2.0-16.2.6. I imagine nobody is on those releases anymore, and they should be moving completely off pacific now anyway.

Reply:
Perfect